### DISCUSSION TITLE
Is there a way to refresh the server certificate?

### DISCUSSION THREAD

**Discussion by cleboo** [cloudnative-pg/cloudnative-pg#803] [Q&A] [answered]
Hi!

After adding a serverAltDNSName to a cluster, I would like cnpg to refresh the server certificate. Is there a way to do this? A `kubectl cnpg reload` seems to have no effect...

Thanks!

**Comment by baurmatt** [comment, marked as answer]
See https://github.com/cloudnative-pg/cloudnative-pg/discussions/7813
